src/index.ts

```diff
@@ -213,19 +214,65 @@ server.tool(
   },
 );
 
+server.tool(
+  "get_follower_ids",
+  "Gets the list of User IDs of users who have added your LINE Official Account as a friend.",
+  {
+    start: z
+      .string()
+      .optional()
+      .describe(
+        "Continuation token for pagination. If omitted, fetches from the beginning.",
+      ),
+    limit: z
+      .number()
+      .int()
+      .min(1)
+      .max(1000)
+      .optional()
+      .describe(
+        "Maximum number of user IDs to retrieve (1-1000). Default is 1000.",
+      ),
+  },
+  async ({ start, limit }) => {
+    try {
+      const params = new URLSearchParams();
+      if (limit) params.append("limit", limit.toString());
+      if (start) params.append("start", start);
+      const res = await axios.get("https://api.line.me/v2/bot/followers/ids", {
+        params,
+        headers: {
+          Authorization: `Bearer ${channelAccessToken}`,
+        },
+      });
+      return createSuccessResponse(res.data);
```

Comment:
```suggestion
      const response = messagingApiClient.getFollowers(start, limit);
      return createSuccessResponse(response);
```
If you use line-bot-sdk-nodejs, the code will be simply and maintainable.